Library/Formula/ffmpeg.rb

```diff
@@ -64,6 +65,7 @@ class Ffmpeg < Formula
   depends_on "libvidstab" => :optional
   depends_on "x265" => :optional
   depends_on "openssl" => :optional
+  depends_on "webp" if build.with? "libwebp"
```

Comment:
I didn't know that (it's my first attempt at tweaking a formula). Will tweak in a second. Thanks :)